server/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "test": "node --test test/**/*.test.js",
     "start": "fastify start -l info app.js",
-    "dev": "fastify start -w -l info -P app.js",
+    "dev": "npx prisma generate && npx prisma migrate dev --name init && fastify start -w -l info -P app.js",
```

Comment:
In reviewing the documentation for these commands, I think we can _remove_ `npx prisma generate` because it will be done as part of `npx prisma migrate dev`.

https://www.prisma.io/docs/orm/reference/prisma-cli-reference#migrate-dev

Reply:
We should also remove the `--name init` from the command, as we shouldn't be generating any new migrations, but just applying any that have not yet been run yet.